src/process.rs

```diff
@@ -0,0 +1,96 @@
+use std::ffi::c_void;
+use std::path::PathBuf;
+
+use ajour_core::fs::PersistentData;
+use serde::{Deserialize, Serialize};
+use winapi::{
+    shared::winerror::WAIT_TIMEOUT,
+    um::{
+        processthreadsapi::{GetCurrentProcess, GetCurrentProcessId, OpenProcess},
+        synchapi::WaitForSingleObject,
+        winbase::QueryFullProcessImageNameW,
+        winnt::{PROCESS_QUERY_LIMITED_INFORMATION, SYNCHRONIZE},
+    },
+};
+
+#[derive(Debug, Serialize, Deserialize)]
+struct Process {
+    pid: u32,
+    name: String,
+}
+
+impl PersistentData for Process {
+    fn relative_path() -> PathBuf {
+        PathBuf::from("pid")
+    }
+}
+
+pub fn avoid_multiple_instances() {
+    if process_already_running() {
+        log::info!("Another instance of Ajour is already running. Exiting...");
+        std::process::exit(0);
+    } else {
+        // Otherwise this is only instance. Save info about this process to the
+        // pid file so future launches of Ajour can detect this running process.
```

Comment:
```suggestion
        // Otherwise this is the only instance. Save info about this process to the
        // pid file so future launches of Ajour can detect this running process.
```